demos/demos_databases_apis/spanner/google_spanner_finance_graph.ipynb

```diff
@@ -0,0 +1,663 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# graphistry register \n",
+    "\n",
+    "# To specify Graphistry account & server, use:\n",
+    "# graphistry.register(api=3, username='...', password='...', protocol='https', server='hub.graphistry.com')\n",
+    "# For more options, see https://pygraphistry.readthedocs.io/en/latest/server/register.html\n",
+    "\n",
+    "graphistry.register(api=3,  \n",
+    "                    protocol  = \"http\", \n",
+    "                    server    = os.getenv(\"GRAPHISTRY_SERVER\"),\n",
+    "                    username  = os.getenv(\"GRAPHISTRY_USERNAME\"),         \n",
+    "                    password  = os.getenv(\"GRAPHISTRY_PASSWORD\"), \n",
+    "                    spanner_config=SPANNER_CONF\n",
+    "                   )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cd1c8aaf-d927-4989-aab2-3a5d0e8858ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the google project id\n",
+    "!gcloud config set project {PROJECT_ID}\n",
+    "%env GOOGLE_CLOUD_PROJECT={PROJECT_ID}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0a03e44e-12e9-4122-b06e-145defc74652",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!gcloud auth application-default login"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f175d952-c1cc-4793-aad5-0cdb5142b6fe",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c2b02e8c-e919-4260-8979-d22f7ac86002",
+   "metadata": {},
+   "source": [
+    "## Example 1: GQL Path Query to Graphistry Visualization of all nodes and edges (LIMIT optional) \n",
+    "\n",
+    "to extract the data from Spanner Graph as a graph with nodes and edges in a single object, a GQL path query is required. \n",
+    "\n",
+    "The format of a path query is as follows, note the p= at the start of the MATCH clause, and the SAFE_TO_JSON(p) without these, \n",
+    "the query will not produce the results needed to properly load a graphistry graph. LIMIT is optional, but for large graphs with millions\n",
+    "        of edges or more, it's best to filter either in the query or use LIMIT so as not to exhaust GPU memory.  \n",
+    "\n",
+    "```sql\n",
+    "GRAPH FinGraph\n",
+    "MATCH p = (a)-[b]->(c) where 1=1 LIMIT 10000 return SAFE_TO_JSON(p) as path\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd3c4b35-e8e1-43c7-bc66-d94b3d3e8451",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query=f'''GRAPH FinGraph\n",
+    "MATCH p = (a)-[b]->(c) where 1=1 {LIMIT_CLAUSE} return SAFE_TO_JSON(p) as path'''\n",
+    "\n",
+    "g = graphistry.spanner_gql_to_g(query)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b544ab6-18b4-4fc9-adb1-3137bec62fb7",
+   "metadata": {},
+   "outputs": [],
```

Comment:
I recommend saving the output of `g.plot()` calls so people can look like in the docs